plugin.yaml

```diff
@@ -2,7 +2,7 @@ plugins:
   libvirt:
     executor: central_deployment_agent
     package_name: cloudify-libvirt-plugin
-    package_version: '0.9.4'
+    package_version: '0.9.5'
 
 data_types:
 
```

Comment:
Do we not need a relationship for the new operations?

Reply:
code was written to handle update on same node, by leveraging some interface like poststart, will check with @mateuszmizer